apps/strapi-cms/src/components/sections/one-trust.json

```diff
@@ -0,0 +1,13 @@
+{
+  "collectionName": "components_sections_one_trusts",
+  "info": {
+    "displayName": "OneTrust"
+  },
+  "options": {},
+  "attributes": {
+    "cdnURL": {
+      "type": "string",
+      "required": true
```

Comment:
@simosalsi What does this `cdnURL` should contain? The name confuses me a bit at the moment

Reply:
What about renaming it to `oneTrustURL` or something like that?

Reply:
It's already inside the `OneTrust` component so it seems redundant imo

Reply:
After a quick call, will update to `oneTrustNoticeURL`